src/components/prompt/prompt.tsx

```diff
@@ -4,18 +4,18 @@ import { type FC } from "react";
 import { styles } from "./styles.js";
 
 interface Props {
-  title: string;
-  info: string;
+  name: string;
+  prompt: string;
```

Comment:
```suggestion
  message: string;
```
"Prompt" is already the name of the component